pollint.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# Copyright 2008 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import os
+
+from lib.linters import Severity as sev
+from lib.linters import build_linters
+from lib.naming import Naming
+from lib.policy import ParsePolicy, _ReadFile
+
+__author__ = 'mikeelkin2@fb.com (Mike Elkin)'
```

Comment:
I forget if I made the comment already on this PR or #79 but I plan on removing author tags as they are not really useful. Commit history gives a much better idea of contributions. Feel free to leave it, this is mostly an FYI